lexicons/tools/ozone/setting/removeOptions.json

```diff
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.setting.removeOptions",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Delete settings by key",
+      "input": {
```

Comment:
I'd recommend also having an `output`, even if it has an empty schema.  We found that we wish we had done that on all lexicons w/ empty output!